sklearn/feature_extraction/tests/test_text.py

```diff
@@ -892,6 +892,19 @@ def test_countvectorizer_vocab_dicts_when_pickling():
         assert_equal(cv.get_feature_names(), unpickled_cv.get_feature_names())
 
 
+def test_countvectorizer_string_object_as_input():
```

Comment:
perhaps it would be better to use `assert_raises` here? see https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tree/tests/test_tree.py#L496 for an example of how it is used.